### PR TITLE
Fixes and and tests for issues #23, merging code from PR #19

### PR DESCRIPTION
**Pull request by TravisEz13**
Will follow up with fixes later, verifying the tests find the problem first

### CODE REVIEW

**Comment by TravisEz13** [comment]
Merged PR #19

**Comment by TravisEz13** [comment]
Fixes #23

File: tests/unit/MSFT_xDisk.tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            context 'Online Unformated disk' {
+                Mock Get-WmiObject -mockwith {return $global:mockedWmi}
+                Mock Get-Disk -mockwith {return $global:mockedDisk0Raw} -verifiable
+                Mock Get-Partition -mockwith {
+                    return $Global:mockedPartition
+                } 
+                
+                Mock Get-Volume -mockwith {
+                    return $global:mockedVolume
+                }
```
Comment (zhenl-microsoft) [inline tests/unit/MSFT_xDisk.tests.ps1]: Better to have unified format (all in one line or all separated into three lines). Same for above places.

**Comment by zhenl-microsoft** [comment]
Reviewed. Looks good to me.

**Pull request merged**